### PR TITLE
refactor plugin-modify-response : Finishing method.

### PR DESCRIPTION
**Pull request by en-o**
// Describe your PR here; eg. Fixes #issueNo

no issueNo,  When I looked at modify-response function today, I found that the author of this code had not finished finishing  

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor/).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-plugin/shenyu-plugin-modify-response/src/main/java/org/apache/shenyu/plugin/modify/response/ModifyResponsePlugin.java [hunk 1]
```diff
@@ -175,14 +175,6 @@ public Boolean skip(final ServerWebExchange exchange) {
...
-            if (!CollectionUtils.isEmpty(handle.getReplaceBodyKeys())) {
-                handle.getReplaceBodyKeys().forEach(info -> {
-                    context.renameKey(info.getPath(), info.getKey(), info.getValue());
-                });
-            }
-            if (!CollectionUtils.isEmpty(handle.getRemoveBodyKeys())) {
-                handle.getRemoveBodyKeys().forEach(context::delete);
-            }
             return context.jsonString();
         }
```
Comment (dengliming) [inline shenyu-plugin/shenyu-plugin-modify-response/src/main/java/org/apache/shenyu/plugin/modify/response/ModifyResponsePlugin.java:179]: How about remove `private void operation(final DocumentContext context, final ModifyResponseRuleHandle handle)` and refactor like this?
```
private String operation(final String jsonValue, final ModifyResponseRuleHandle handle) {
            DocumentContext context = JsonPath.parse(jsonValue);
            if (!CollectionUtils.isEmpty(handle.getAddBodyKeys())) {
                handle.getAddBodyKeys().forEach(info -> context.put(info.getPath(), info.getKey(), info.getValue()));
            }
            if (!CollectionUtils.isEmpty(handle.getReplaceBodyKeys())) {
                handle.getReplaceBodyKeys().forEach(info -> context.renameKey(info.getPath(), info.getKey(), info.getValue()));
            }
            if (!CollectionUtils.isEmpty(handle.getRemoveBodyKeys())) {
                handle.getRemoveBodyKeys().forEach(context::delete);
            }
            return context.jsonString();
        }
```

Comment (en-o) [reply shenyu-plugin/shenyu-plugin-modify-response/src/main/java/org/apache/shenyu/plugin/modify/response/ModifyResponsePlugin.java:179]: Idea prompted me. I thought the line was too long after I changed it, so I changed it back. 😅 (changed)

Comment (dengliming) [reply shenyu-plugin/shenyu-plugin-modify-response/src/main/java/org/apache/shenyu/plugin/modify/response/ModifyResponsePlugin.java:179]: IMO, You don't need to overload one more method `private void operation(final DocumentContext context, final ModifyResponseRuleHandle handle)`.

Comment (en-o) [reply shenyu-plugin/shenyu-plugin-modify-response/src/main/java/org/apache/shenyu/plugin/modify/response/ModifyResponsePlugin.java:179]: Does that mean just like it used to be? Then I'll turn it off.
> 意思是就像原来那样吗？那我关了(是我看到这段觉得别扭所以给他整一起了)

**Pull request closed without merging**